python/meterpreter/ext_server_stdapi.py

```diff
@@ -1146,11 +1146,8 @@ def stdapi_sys_process_execute(request, response):
     flags = packet_get_tlv(request, TLV_TYPE_PROCESS_FLAGS)['value']
     if len(cmd) == 0:
         return ERROR_FAILURE, response
-    if os.path.isfile('/bin/sh'):
-        args = ['/bin/sh', '-c', cmd + ' ' + raw_args]
-    else:
-        args = [cmd]
-        args.extend(shlex.split(raw_args))
```

Comment:
There wasn't much context on why this was handled differently in the past, but here's the commit that added it https://github.com/rapid7/metasploit-payloads/commit/5c828d01ebef059e161dee6785e2b8bc5bea8a37